src/Controllers/MovieController.cs

```diff
@@ -0,0 +1,40 @@
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Localization;
+using Microsoft.EntityFrameworkCore;
+using TrainingNet.Models.DataBase;
+using TrainingNet.Repositories;
+using TrainingNet.Repositories.Database;
+using TrainingNet.Repositories.Interfaces;
+
+namespace TrainingNet.Controllers
+{
+    [Route("[controller]")]
+    public class MovieController : Controller
+    {
+        private readonly IHtmlLocalizer<HomeController> _localizer;
```

Comment:
This service is not used in this controller (at least not yet :stuck_out_tongue: )

Reply:
exactly, not yet. I mean, I *should* be using it, but I'm lazy.